dev/com.ibm.ws.webcontainer.servlet.4.0_fat.2/fat/src/com/ibm/ws/fat/wc/FATSuite.java

```diff
@@ -0,0 +1,87 @@
+/*******************************************************************************
+ * Copyright (c) 2012, 2024 IBM Corporation and others.
```

Comment:
I just copied this from the original bucket and modified it so left the copyright header in tact